devenv.nix

```
@@ -142,6 +142,73 @@
       EOF
     '';
   };
+  scripts."devenv-generate-individual-docs" = {
+    description = "Generate individual docs of all devenv modules";
+    exec = ''
+      mkdir -p docs/{autogen-language-docs,autogen-service-docs,autogen-process-manager-docs}
+
+      nix build --impure --extra-experimental-features 'flakes nix-command' --show-trace --print-out-paths '.#devenv-generate-individual-docs'
+      cp -r result/docs/individual-docs/* docs/
+      chmod -R u+rwX docs/{autogen-language-docs,autogen-service-docs,autogen-process-manager-docs}
```

I think this should have the same effect:
```suggestion
      cp -r --no-preserve=all result/docs/individual-docs/* docs/
```